src/Routers/FunctionsRouter.js

```diff
@@ -119,10 +119,11 @@ export class FunctionsRouter extends PromiseRouter {
         if (typeof message === 'string') {
           return reject(new Parse.Error(code, message));
         }
+        const error = new Parse.Error(code, message.message || message);
```

Comment:
Thanks for the PR! I'm only afraid of `message` being null/undefined for any reason. So I'd suggest `const error = new Parse.Error(code, message && message.message || message);`